enumablelist.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+
+p(list.all? { |item| item < 5 })
+
+p(list.all? { |item| item > 5 })
+
+p(list.any? { |item| item == 2 })
+
+p(list.any? { |item| item == 5 })
+
+p(list.filter(&:even?))
```

Comment:
- [ ] OPTIONAL
Change `filter` to f`ilter?`, for the app to run without errors.